api/src/controllers/young/index.js

```diff
@@ -464,6 +465,58 @@ router.put("/", passport.authenticate("young", { session: false, failWithError:
   }
 });
 
+router.put("/:id/change-cohort/", passport.authenticate("young", { session: false, failWithError: true }), async (req, res) => {
+  try {
+    const { error, value } = validateYoung(req.body, req.user);
+    if (error) return res.status(400).send({ ok: false, code: ERRORS.INVALID_PARAMS, error: error.message });
+
+    const young = await YoungObject.findById(req.user._id);
+    if (!young) return res.status(404).send({ ok: false, code: ERRORS.NOT_FOUND });
+    const oldSessionPhase1Id = young.sessionPhase1Id;
+    young.set({ sessionPhase1Id: "" });
```

Comment:
peut etre supprimer le `meetingPointId` aussi ?
```suggestion
    young.set({ sessionPhase1Id: undefined });
```